hasher-matcher-actioner/webapp/src/Upload.js

```diff
@@ -7,41 +7,57 @@ import { Link } from 'react-router-dom';
 import { Button, Col, Collapse, Image, Row } from 'react-bootstrap';
 import Spinner from 'react-bootstrap/Spinner';
 
+import { postAPI } from './Api'
+
 export default function Upload() {
-  const [ fileName, setFileName ] = useState('Select Browse to choose a file.');
-  const [ submitting, setSubmitting ] = useState(false);
-  const [ submitted, setSubmitted ] = useState(false);
+  const [fileName, setFileName] = useState('Select Browse to choose a file.');
+  const [submitting, setSubmitting] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+  const [image, setImage] = useState({ preview: "", raw: "" });
+
+  function handleUpload() {
```

Comment:
nit: I like my './Api' equivalent to have more function oriented surfaces. 

eg.
```
import { uploadToHashSourceBucket } from './Api';

...
  uploadToHashSourceBucket(fileObject).then(...);
...

The present setup 'leaks' the internal details to the UI component and is nastier to reuse.